docs/reference/commandline/ps.md

```diff
@@ -50,6 +50,8 @@ The currently supported filters are:
 * name (container's name)
 * exited (int - the code of exited containers. Only useful with `--all`)
 * status (created|restarting|running|paused|exited)
+* ancestor (image id) - filters containers that were created from the
```

Comment:
we probably should avoid using "image id" here, because the filter also accepts "image name", right?

Will need some examples, to illustrate/describe the possible options